src/main/kotlin/com/openlattice/linking/controllers/RealtimeLinkingController.kt

```diff
@@ -52,6 +56,33 @@ class RealtimeLinkingController(
         return linkableEntitySets.minus(entitySetsNeedLinking)
     }
 
+    @RequestMapping(
+            path = [RealtimeLinkingApi.MISSING],
+            method = [RequestMethod.POST],
+            produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    override fun getEntitiesMissingLinking(@RequestBody entitySetIds: Set<UUID>): Map<UUID, Set<UUID>> {
+        ensureAdminAccess()
+        val linkableEntitySets = lqs
+                .getLinkableEntitySets(linkableTypes, entitySetBlacklist, whitelist.orElse(setOf()))
+                .toSet()
+        val linkableRequestedEntitySets = linkableEntitySets
+                .intersect(entitySetIds)
+        val nonLinkableRequestedEntitySets = entitySetIds.subtract(linkableEntitySets)
+        if(nonLinkableRequestedEntitySets.isNotEmpty()) {
+            logger.warn("The following entitySets were queried for entities not yet linked, " +
+                    "but are not linkable: $nonLinkableRequestedEntitySets")
+        }
+
+        val entitiesNeedLinking = lqs
```

Comment:
Really minor, but you could just return the result of this and not assign it to a variable